### PR TITLE
Add shields for states of Mexico

### LINKED ISSUE #404: Shields of Mexico
**(EN)**

Federal Highways of Mexico have a unique black-on-white shield design. Almost all states of Mexico use a similar design with a square top for their State Highways.

Many numbered routes in Mexico are covered by `route=road` relations, but `network=*` tagging is limited. Values of `network=*` in Mexico are country-prefixed, but it is unclear which values correspond to Federal Highways, highways of Mexico City, and highways of the State of Mexico. A decision would need to be made on whether this tagging scheme is sufficient, and whether it is appropriate mapper feedback to support it.

**(ES)**

Las Carreteras Federales de México tienen un diseño único de escudo en blanco y negro. Casi todos los estados de México utilizan un diseño similar con una parte superior cuadrada para sus Carreteras Estatales.

Muchas rutas numeradas en México están cubiertas por las relaciones `route=road`, pero el etiquetado `network=*` es limitado. Los valores de `network=*` en México tienen el prefijo del país, pero no está claro qué valores corresponden a Carreteras Federales, carreteras de la Ciudad de México y carreteras del Estado de México. Sería necesario tomar una decisión sobre si este esquema de etiquetado es suficiente y si este soporte brindará una buena retroalimentación a los mapeadores.

----

Federal Highway shield:
Rótulo de las carreteras federales:
Carretera_federal_45 Carretera_federal_2D

State Highway shield:
Rótulo de las carreteras estatales:
Carretera_estatal_176_(Sonora)

### PR DESCRIPTION
**Pull request by claysmalley**
Fixes #404

This does not include shields for the State of Mexico, because there are two competing abbreviations for the state (EM, MEX). Waiting on the local mapping community to come to a consensus.

---

No incluye escudos para el Estado de México, porque existe dos abreviaturas que compiten para el estado (EM, MEX). Esperando a que la comunidad cartográfica local llegue a un consenso.

### CODE REVIEW

**Comment by ZeLonewolf** [comment]
Jalisco state:

image

Of note, there is only a single route, in the 3-digit variety, so we have no example of what the 2-digit version looks like.

**Comment by ZeLonewolf** [comment]
Looks like there is some work still to remove network names from route relation refs:

image

File: src/js/shield_defs.js [hunk 1]
```diff
@@ -590,16 +591,6 @@ export function loadShields() {
         },
       })
   );
-  shields["MX:JAL"] = {
```
Comment (1ec5) [inline src/js/shield_defs.js]: In that case, we should keep #411 open rather than declaring it fixed, to allow for more evidence to be gathered.

**Review by ZeLonewolf — APPROVED** [review]
LGTM, sounds like we've resolved the outstanding issues.

**Pull request merged**